src/database/Database.cpp

```diff
@@ -380,7 +380,7 @@ Database::clearPreparedStatementCache()
 }
 
 void
-Database::initialize()
+Database::initialize(bool dropBuckets)
```

Comment:
I think the contract would be more natural if it was instead a `keepBuckets`

Reply:
or even `enum Buckets { DROP_BUCKETS, KEEP_BUCKETS }`

Reply:
I think I'll actually move it out to the main caller in Application; it's not really ideal to have Database calling anything in BucketManager.